examples/webgl_octree.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<title>three.js webgl - octree </title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+	<link type="text/css" rel="stylesheet" href="main.css">
+</head>
+
+<body>
+	<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js",
+					"three/addons/": "./jsm/"
+				}
+			}
+		</script>
+
+	<script type="module">
+
+		import * as THREE from 'three';
+
+		import Stats from 'three/addons/libs/stats.module.js';
+		import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
+		import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+		import { Octree } from 'three/addons/math/Octree.js';
+		import { OctreeHelper } from 'three/addons/helpers/OctreeHelper.js';
+
+		let camera, scene, renderer, controls, stats;
+		let octree, octreeHelper;
+
+		let mesh;
+		const positions = [];
+		const count = 10000;
+
+		const mouse = new THREE.Vector2( 1, 1 );
+
+		const color = new THREE.Color();
+		const redColor = new THREE.Color( 1, 0, 0 );
+
+		init();
+		animate();
+
+		function init() {
+
+			camera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 0.1, 2000 );
+			camera.position.set( 0, 0, 100 );
+			camera.lookAt( 0, 0, 0 );
+
+			scene = new THREE.Scene();
+
+			const light = new THREE.HemisphereLight( 0xffffff, 0x888888, 3 );
+			light.position.set( 0, 1, 0 );
+			scene.add( light );
+
+
+			octree = new Octree();
+
+
+
+			const geometry = new THREE.IcosahedronGeometry( 0.15, 3 );
+			const material = new THREE.MeshPhongMaterial( { color: 0xffffff } );
+
+			mesh = new THREE.InstancedMesh( geometry, material, count );
+			const matrix = new THREE.Matrix4();
+			for ( let i = 0; i < count; i ++ ) {
+
+				const point = new THREE.Vector3( Math.random() * 100 - 50, Math.random() * 100 - 50, 0 );
+				matrix.setPosition( point );
+				mesh.setMatrixAt( i, matrix );
+				mesh.setColorAt( i, color );
+
+				// add octree triangle
+				const triangle = new THREE.Triangle( point, point, point );
+				triangle.index = i;
+				octree.addTriangle( triangle );
```

Comment:
I would expect you setup the `Octree` like so:
```js
octree.fromGraphNode( scene );
```
Normally, spatial data structures are not setup with raw (degenerated) triangles.

Reply:
Okay, I've modified it to use fromGraphNode